modules/select.rb

```diff
@@ -1,13 +1,17 @@
 require_relative '../app'
+require_relative 'read_data'
+require 'pry'
 
```

Comment:
- [ ] Same problem raised here. Please fix that.

Reply:
Hi @Omar-Muhamad,
We've seen the error you are talking about and we are going to work on that 
Thank you.